tests/conftest.py

```diff
@@ -577,3 +577,50 @@ def create_gcs_bucket(bucket_name: str):
 
     gcs_store = GcsStore(name=bucket_name, source="")
     return gcs_store
+
+
+# ----------------- Docker -----------------
+
+
+def run_shell_command(subprocess, cmd: list[str]):
+    # Run the command and wait for it to complete
+    result = subprocess.run(cmd, capture_output=True, text=True)
+
+    if result.returncode != 0:
+        print(result.stdout)
+
+    # Check for success
+    assert result.returncode == 0
+
+
+@pytest.fixture
+def local_docker():
```

Comment:
can we make use of this fixture as part of a unittest somewhere? maybe incorporate with some of the checks made in `dockerize.py`?

Reply:
Done

Reply:
This is now used in: https://github.com/run-house/runhouse/blob/b1ae55ed21c9049c9c5a7530d2c556dfe7b3883a/tests/test_http_server.py